test/test-async.js

```diff
@@ -2806,20 +2999,20 @@ exports['cargo bulk task'] = function (test) {
 };
 
 exports['cargo drain once'] = function (test) {
-   
+
```

Comment:
I'm personally a fan of more vertical whitespace but these don't match the rest of the file and they seem extraneous to the work done in this PR. :)